bleach/sanitizer.py

```diff
@@ -8,21 +8,23 @@
 from bleach import parse_shim
 
 
-#: List of allowed tags
-ALLOWED_TAGS = [
-    "a",
-    "abbr",
-    "acronym",
-    "b",
-    "blockquote",
-    "code",
-    "em",
-    "i",
-    "li",
-    "ol",
-    "strong",
-    "ul",
-]
+#: Set of allowed tags
+ALLOWED_TAGS = frozenset(
```

Comment:
I thought about going through and making the other constant values frozensets where applicable, too. Seems like a good idea. Thoughts?

Reply:
👍 sgtm. I think we used lists for backwards compat or as a holdover from Python 2 support.

Reply:
I'll look at doing that in another PR.

Reply:
I think the set is a better data structure and we're bumping the major version. I was just trying to recall context on why it was a list.